Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -1697,7 +1697,7 @@ bool Renderer::UseVertexDepthRange() const
     return false;
 
   // We need a full depth range if a ztexture is used.
-  if (bpmem.ztex2.type != ZTEXTURE_DISABLE && !bpmem.zcontrol.early_ztest)
+  if (bpmem.ztex2.op != ZTexOp::Disabled && !bpmem.zcontrol.early_ztest)
```

Comment:
This seems to have been a typo originally; `type` was the format (previously `TEV_ZTEX_TYPE_U8`/`_U16`/`_U24`).  I'm not sure if that affected anything though.